tests/foreman/virtwho/test_cli_virtwho.py

```diff
@@ -275,18 +276,44 @@ def test_positive_filter_option(self):
             'filtering-mode': 'whitelist',
             'whitelist': regex,
         }
+        blacklist = {
+            'id': vhd['id'],
+            'filtering-mode': 'blacklist',
+            'blacklist': regex,
+        }
+        if self.hypervisor_type == 'esx':
+            whitelist['filter-host-parents'] = regex
+            blacklist['exclude-host-parents'] = regex
+        # Update Whitelist and check the result
         VirtWhoConfig.update(whitelist)
         result = VirtWhoConfig.info({'id': vhd['id']})
-        self.assertEquals(result['connection']['filtering'], 'Whitelist')
-        self.assertEquals(result['connection']['filtered-hosts'], regex)
+        self.assertEqual(result['connection']['filtering'], 'Whitelist')
+        self.assertEqual(result['connection']['filtered-hosts'], regex)
+        if self.hypervisor_type == 'esx':
+            self.assertEqual(result['connection']['filter-host-parents'], regex)
+        config_file = get_configure_file(vhd['id'])
+        command = get_configure_command(vhd['id'])
+        deploy_configure_by_command(command)
+        self.assertEqual(get_configure_option('filter_hosts', config_file), regex)
+        if self.hypervisor_type == 'esx':
+            self.assertEqual(get_configure_option('filter_host_parents', config_file), regex)
+        # Update Blacklist and check the result
+        VirtWhoConfig.update(blacklist)
+        result = VirtWhoConfig.info({'id': vhd['id']})
+        self.assertEqual(result['connection']['filtering'], 'Blacklist')
+        self.assertEqual(result['connection']['excluded-hosts'], regex)
+        if self.hypervisor_type == 'esx':
+            self.assertEqual(result['connection']['exclude-host-parents'], regex)
         config_file = get_configure_file(vhd['id'])
         command = get_configure_command(vhd['id'])
```

Comment:
`config_file` and `command ` should be not changed for `filter_host_parents` and `exclude-host-parents` validation.

Reply:
ok, I will remove it